tests/python/unittest/test_operator.py

```diff
@@ -5158,6 +5158,80 @@ def check_squeeze_op(shape, axis=None):
     test = mx.sym.squeeze(data, axis=(2, 4))
     check_numeric_gradient(test, [data_tmp])
 
+def py_adaptive_avg_pool(x, height, width):
+    # 2D per frame adaptive avg pool
+    def adaptive_avg_pool_frame(x, y):
+        isizeH, isizeW = x.shape
+        osizeH, osizeW = y.shape
+        for oh in range(osizeH):
+            istartH = int(np.floor(1.0 * (oh * isizeH) / osizeH))
+            iendH = int(np.ceil(1.0 * (oh + 1) * isizeH / osizeH))
+            kH = iendH - istartH
+            for ow in range(osizeW):
+                istartW = int(np.floor(1.0 * (ow * isizeW) / osizeW))
+                iendW = int(np.ceil(1.0 * (ow + 1) * isizeW / osizeW))
+                kW = iendW - istartW
+                xsum = 0
+                for ih in range(kH):
+                    for iw in range(kW):
+                        xsum += x[istartH+ih][istartW+iw]
+                y[oh][ow] = xsum / kH / kW
+
+    B,C,_,_ = x.shape
+    y = np.empty([B,C,height, width], dtype=x.dtype)
+    for b in range(B):
+        for c in range(C):
+            adaptive_avg_pool_frame(x[b][c], y[b][c])
+    return y
+
+@with_seed()
+def test_adaptive_avg_pool_op():
+    def check_adaptive_avg_pool_op(shape, output_size):
+        x = mx.nd.random.uniform(shape=shape)
+        y = mx.nd.AdaptiveAvgPool2D(x, output_size=output_size)
+        npy = py_adaptive_avg_pool(x.asnumpy(), output_size, output_size)
+        assert_almost_equal(y.asnumpy(), npy)
+    shape = (2, 2, 10, 10)
+    for i in range(1, 11):
+        check_adaptive_avg_pool_op(shape, i)
+
+def py_bilinear_resize(x, outputHeight, outputWidth):
```

Comment:
Considering this method is only being used by one test, please define this inside test_bilinear_resize_op to avoid confusion.